Header header
int32 TransmitterID
float32 AgentPosX
float32 AgentPosY
float32 AgentTheta
float32 VirtualCenterX
float32 VirtualCenterY
float32 V_BLF
